casepro/statistics/models.py

```diff
@@ -230,6 +230,9 @@ def _get_count_set(cls, item_type, scopes):
 
     class Meta:
         index_together = ("item_type", "scope")
+        indexes = [
+            Index(name="stats_totalcount_unsquashed", fields=("item_type", "scope"), condition=Q(is_squashed=False))
```

Comment:
@nicpottier @norkans7 @ericnewcomer just a heads up that this became a thing in Django 2.2.. nice way to create simple partial indexes.